### DISCUSSION TITLE
add Instance ID -> FCM getToken migration guide on changelog

### DISCUSSION THREAD

**Discussion by efstathiosntonas** [invertase/react-native-firebase#5338] [General]
@mikehardy it would be nice to add the migration link to CHANGELOG and/or docs, will save some time digging for it: https://firebase.google.com/docs/projects/manage-installations#fid-iid

**Comment by mikehardy** [comment]
great idea thanks for fishing it out! I just made the change

**Reply by efstathiosntonas** [reply]
Thanks @mikehardy keep it up 😀
